.github/workflows/operator-ci.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           kubernetes version: ${{ env.KUBERNETES_VERSION }}
           github token: ${{ secrets.GITHUB_TOKEN }}
           driver: docker
-          start args: --addons=ingress --memory=${{ env.MINIKUBE_MEMORY }}
+          start args: --addons=ingress --memory=${{ env.MINIKUBE_MEMORY }} --cni cilium --cpus=max
```

Comment:
My only concern here would be if Cilium affects somehow the tests/cluster stability. Maybe we could try and run a few tens of Operator CI runs to be sure it doesn't introduce new flakiness. WDYT?

Reply:
Is there a trick to do this or do I need to manually click `re-run jobs`? :smile:

Reply:
Yes, there's a trick. 🙂 See e.g. https://github.com/vmuzikar/keycloak/blob/85bf6a65d1686a6ab208c8c8ba5e3b47bb55755f/.github/workflows/ci.yml#L77-L78

Reply:
Checking the stability is not a blocker from my side, just something nice to have.